config/modules/dhcp.conf

```diff
@@ -0,0 +1,14 @@
+# Tests to build.
+build docker/modules
+
+# Default built-in tests.
+add pass
```

Comment:
These files should build/include each other as necessary, not repeat information.  E.g., "ping" has nothing to do with dhcp in this context.  Something about this (and the modules/many.conf one) don't "smell" right...